src/player.h

```diff
@@ -1214,6 +1214,9 @@ class player : public Character, public JsonSerializer, public JsonDeserializer
             turn_died = (turn_died != -1) ? turn : turn_died;
         }
 
+        // are we looking around via ';'?
+        bool is_looking_around;
```

Comment:
While I like a solution to that problem, this solution is really ugly (a public member just for one task). It should probably be in `game` as it does not really concern the player object. Or you could add it as a parameter to `game::draw_ter`, it is called directly from `game::look_around`, the only place where the non-default value would be needed.